docusaurus-docs/community/contribute/local-setup-ui.md

```diff
@@ -9,46 +9,62 @@ To get started with conda-store-ui development, there are a couple of options. T
+For more advanced development on conda-store-ui, the first option might not be sufficient. If you need to work extensively in the UI codebase, then you will probably want to run the UI web app locally rather than within a Docker container.
 
-### Set up your environment
+Note: this setup still uses Docker to run the rest of the Conda Store stack. That means that the Conda Store database, server, worker, and storage services will all run in Docker containers. But the frontend web app (conda-store-ui) will run locally (not in a Docker container) with this setup.
+
+### Set up your Conda environment
 
 This project uses [Conda](https://conda.io) for package management. To set up Conda, please see their [installation documentation](https://docs.conda.io/projects/conda/en/latest/user-guide/install/index.html).
-1. Change to the project root ` cd conda-store-ui`
-2. From the project root create the conda environment `conda env create -f environment_dev.yml`
-3. Activate the development environment `conda activate cs-ui-dev-env`
-4. Install yarn dependencies `yarn install`
 
-### Run the application
+1. Change to the project root: `cd conda-store-ui`
+2. From the project root, create the conda environment: `conda env create -f environment_dev.yml`
+
+### Run conda-store-ui locally (and the rest of the stack in Docker)
+
+Tip: Make sure you are in the project root: `cd conda-store-ui`
 
-1. Run `yarn run start` and wait for the application to finish starting up
-Open your local browser and go to [http://localhost:8000](http://localhost:8000) so see conda-store-ui.
-3. You can then log in using the default username of `username` and default password of `password`.
+1. Activate the development environment: `conda activate cs-ui-dev-env`
+2. Set environment variables. Copy `.env.example` to `.env`. All default settings should work, but if you want to test against a different version of conda-store-server, you can specify it in the `.env` file by setting the `CONDA_STORE_SERVER_VERSION` variable to the desired version.
+3. Set the package manager to match package.json (`packageManager`): `corepack enable`
+4. Install/update JavaScript dependencies: `yarn install`
+5. Run `yarn run start` and wait for the application to finish starting up. This command will run a local dev server for the UI app and run the other Conda Store services in Docker.
+6. Open your local browser and go to [http://localhost:8000](http://localhost:8000) to see conda-store-ui.
+7. You can then log in using the default username of `username` and default password of `password`.
 
 **Note:** Hot reloading is enabled, so when you make changes to source files, your browser will reload and reflect the changes.
 
 ## Run the test suite
 
-We currently use jest in order to run unit tests.
+We currently use Jest in order to run unit tests.
 
 ```bash
 yarn test     // find every test with the .test.[tsx|ts] extension
```

Comment:
yarn is also still used down here. Not sure about getting rid of all mention of installing yarn?

Reply:
If one creates the conda environment per the instructions above yarn should be already installed

Reply:
This section should probably somehow be moved into Advanced development then.